CybridSDK/Utilities/Networking/CybridJSONDecoder.swift

```diff
@@ -11,6 +11,7 @@ import Foundation
 
 final class CybridJSONDecoder: JSONDecoder {
 
+    // swiftlint:disable:next cyclomatic_complexity
```

Comment:
Why do we have to do the decoding manually for all models? I know we had to do it for models where large numbers were coming back, but the deserialize that the OpenAPI generated code generates should work for the other models.

Reply:
Yes this is a horrible problem, but the Date and custom enums are bugged so if we don't decode manually it didn't works!